spec/spec.md

```diff
@@ -119,22 +123,22 @@ This section consolidates in one place common terms used across open standards t
 [[def:Self Issued OpenID Provider (SIOP), Self Issued OpenID Provider, SIOP]]  
 ~ An OpenID Provider (OP) used by an [[ref:End User]] to prove control over a cryptographically verifiable identifier such as a [[ref:DID]].
 
-[[def:Verifiable Credential (VC), Verifiable Credential, VC]]
+[[def:Verifiable Credential (VC), Verifiable Credential, VC, Verifiable Credentials]]
```

Comment:
why is this enumeration needed? what does it do?

Reply:
this one helps my link in the first paragraph... I do the first link/ref with plural Verifiable Credentials.